build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1724,6 +1757,13 @@ message CacheCapabilities {
   // Note that this does not imply which if any compressors are supported by
   // the server at the gRPC level.
   repeated Compressor.Value supported_compressors = 6;
+
+  // Compressors supported for inlined data in messages uploaded to the
+  // server like
+  // [BatchUpdateBlobsRequest][build.bazel.remote.execution.v2.BatchUpdateBlobsRequest],
+  // [ActionResult][build.bazel.remote.execution.v2.ActionResult] and
+  // [OutputFile][build.bazel.remote.execution.v2.OutputFile].
+  repeated Compressor.Value supported_inline_compressors = 7;
```

Comment:
supported_upload_compressors?
Considering that we're no longer doing inlining for uploads

Reply:
I think it might be better to be more specific than `supported_upload_compressors`. How about `supported_batch_compressors` ? Even though it's only strictly required to know in order to use compressed data in `BatchUpdateBlobsRequest`, the same values will probably be the ones selected from for `BatchReadBlobsResponse`.

And what do you think about renaming `supported_compressors` to `supported_bytestream_compressors` ?